pragma-sdk/pragma_sdk/supported_assets.yaml

```diff
@@ -103,3 +103,25 @@
   ticker: 'EKUBO'
   coingecko_id: 'ekubo-protocol'
   starknet_address: '0x075afe6402ad5a5c20dd25e10ec3b3986acaa647b77e4ae24b0cbc9a54a27a87'
+
+- name: 'Toncoin'
```

Comment:
Also misses all the currencies referenced here:
https://docs.pragma.build/api/off-chain-feeds/supported-assets